index.html

```diff
@@ -52,6 +54,164 @@ <h1> Hey There. <br> I'm Dennis <br>
+                  <div class="projectBtn">html</div>
+                  <div class="projectBtn">bootstrap</div>
+                  <div class="projectBtn">Ruby</div>
+                </div>
+              </div>
+              <span class="actionBtn">See Project</span>
+            </div>
+          </div>
+
+          <div id="cardWork">
+            <div class="bg">
+              <div class="prjectDescription">
+                <span>Profesional Art <br/> Printing Data</span>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="projectBtnGroup">
+                  <div class="projectBtn">html</div>
+                  <div class="projectBtn">bootstrap</div>
+                  <div class="projectBtn">Ruby</div>
+                </div>
+              </div>
+              <span class="actionBtn">See Project</span>
+            </div>
+          </div>
+
+          <div id="cardWork">
+            <div class="bg">
+              <div class="prjectDescription">
+                <span>Profesional Art <br/> Printing Data</span>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="projectBtnGroup">
+                  <div class="projectBtn">html</div>
+                  <div class="projectBtn">bootstrap</div>
+                  <div class="projectBtn">Ruby</div>
+                </div>
+              </div>
+              <span class="actionBtn">See Project</span>
+            </div>
+          </div>
+
+          <div id="cardWork">
+            <div class="bg">
+              <div class="prjectDescription">
+                <span>Profesional Art <br/> Printing Data</span>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="projectBtnGroup">
+                  <div class="projectBtn">html</div>
+                  <div class="projectBtn">bootstrap</div>
+                  <div class="projectBtn">Ruby</div>
+                </div>
+              </div>
+              <span class="actionBtn">See Project</span>
+            </div>
+          </div>
+
+          <div id="cardWork">
+            <div class="bg">
+              <div class="prjectDescription">
+                <span>Profesional Art <br/> Printing Data</span>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="projectBtnGroup">
+                  <div class="projectBtn">html</div>
+                  <div class="projectBtn">bootstrap</div>
+                  <div class="projectBtn">Ruby</div>
+                </div>
+              </div>
+              <span class="actionBtn">See Project</span>
```

Comment:
- [ ] For The button (e.g., "See project") Kindly I recommend you to use `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design),it will make your HTML code semantically correct